### DISCUSSION TITLE
Enhancement: Search products by SKU

### DISCUSSION THREAD

**Discussion by Wikisaqui** [addonify/addonify-compare-products#187] [General]
In the search modal, products should be searchable by SKU as well.

**Comment by h1dd3nsn1p3r** [comment]
Hi @Wikisaqui

We will explore this possibility shortly. 

Please stay tuned 👋

**Reply by Wikisaqui** [reply]
Thanks @h1dd3nsn1p3r.

If I post the code to do so here would that be helpful/speed up the process?
Or is there a way I can contribute to the project directly?

Thanks

**Reply by h1dd3nsn1p3r** [reply]
Hi @Wikisaqui 

That's a good idea. This is an open-source project. Please fork the project & create a PULL request. Make sure you push that in this repo too.
